From be34c91afb1b66654bd91b164f26b7feac4cfd58 Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Mon, 3 Nov 2014 15:59:26 +0100
Subject: [LIBREPORT PATCH] ureport: be able to configure ContactEmail from GUI

reporter-rhtsupport reads uReport_ContactEmail configuration option
before attaching the contact email to uReport. It is likely that users
will run the reporter from the GUI, so they must be able to configure
the reporter from the GUI too.

Related to rhbz#1150388

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/plugins/report_uReport.xml.in | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/plugins/report_uReport.xml.in b/src/plugins/report_uReport.xml.in
index 3039baa..6a6c45a 100644
--- a/src/plugins/report_uReport.xml.in
+++ b/src/plugins/report_uReport.xml.in
@@ -20,6 +20,11 @@
             <_note-html>Adds your Bugzilla login to CC List of already reported bugs.</_note-html>
             <default-value>yes</default-value>
         </option>
+        <option type="text" name="uReport_ContactEmail">
+            <_label>Contact email address</_label>
+            <allow-empty>yes</allow-empty>
+            <_description>Email address that can be used by ABRT server to inform you about news and updates</_description>
+        </option>
         <option type="bool" name="uReport_SSLVerify">
             <_label>Verify SSL</_label>
             <_description>Check SSL key validity</_description>
-- 
1.8.3.1

